docs/rfcs/002-archival_storage/README.md

```diff
@@ -530,18 +531,18 @@ Response:
       }
     ]
   },
-  "accessLocation": {
-    "type": "Location",
-    "provider": {
-      "type": "Provider",
-      "id": "aws-s3-ia",
-      "label": "AWS S3 - Infrequent Access"
+  "storageLocations": [
+    {
+      "type": "Location",
+      "provider": {
+        "type": "Provider",
+        "id": "aws-s3-ia",
+        "label": "AWS S3 - Infrequent Access"
+      },
+      "bucket": "bucketname",
+      "path": "/digitised/b24923333",
+      "url": "http://bucketname.s3-eu-west-1.amazonaws.com/digitised/b24923333"
```

Comment:
If I’m a consumer like DLCS or DDS, how do I know which of these locations I should request? Look at the provider?

Reply:
This is how the DDS does it atm:
```
...
bucketName = storageManifest["accessLocation"]["bucket"].Value<string>();
path = storageManifest["accessLocation"]["path"].Value<string>();
...
```

Let me know if this needs changing, it's easily done and redeployed.